packages/fxa-settings/src/models/reliers/browser-relier.ts

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Relier } from "./rellier";
+import { bind, ContextValidation as V } from '../../lib/context';
+import { Context } from "../../lib/context/interfaces/context";
+
+export interface BrowserRelierData {
```

Comment:
This pattern of having a 'data' interface is used across all relier models.

I am thinking these are what our controls use instead of the classes. This should make testing easy, cause you can just passing whatever state is needed. Thereby decoupling the tests that binding works as expected from the tests on the components themselves.